### PR TITLE
Weather app by Amanda Elvkull, Carina Moreno & Ulrika Öhman

### PR DESCRIPTION
**Pull request by uohman**
Collaborators: Amanda Elvkull, Carina Moreno & Ulrika Öhman

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    <link href="https://fonts.google.com/specimen/Montserrat"/>
+    
+
+    <title>Weather app</title>
+</head>
+<h2>What is the weather like today?</h2>
+<body>
+    <!--searchbar for choosing a different city-->
+    <input class= "searchCity" type="text" id="searchCity" placeholder="Search for a city" autofocus>
+    <button onclick="getLocation()" id="button"> Click for Current Location</button>
```
Comment (KajaWilbik) [inline index.html:17]: Congratulation on "getLocation" button, I checked it today, and showed proper weather and proper location.

File: script.js [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+const sunset = new Date(json.sys.sunset *1000);
+    const sunriseShort = sunrise.toLocaleTimeString([], {timeStyle: 'short'}) //declare new variable to show only hh:mm
+    const sunsetShort = sunset.toLocaleTimeString([], {timeStyle: 'short'})
+
+const nameCity = json.name 
+
+console.log (temperature,currentDescription,sunriseShort,sunsetShort,nameCity) //test for console that the function is working. 
+
+ //Print out the API fetch for header
+container.innerHTML=`${currentDescription} | ${temperature}°<br>sunrise ${sunriseShort}<br>sunset ${sunsetShort}`
```
Comment (KajaWilbik) [inline script.js:29]: It is a very nice way to make innerHTML readable and short.

File: script.js [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+      }
+      
+      }
+  )
+  })
+                        
+  } if (navigator.geolocation) {
+           
+  navigator.geolocation.getCurrentPosition((position) =>{
+  var lat = position.coords.latitude; // set lat for api url
```
Comment (KajaWilbik) [inline script.js:168]: what that part of the code is doing?

**Review by KajaWilbik — APPROVED** [review]
Your work is amazing. Everything works, you did extra steps and the design looks exactly how it should be. Small tips: would be nice if the landing page is not completely white at the beginning, because it gives a feeling like something is wrong with the page. Great job!!

File: script.js [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+      <div class="dayAndTemp">
+          <div class="day">${weekDay(filterTable[3].dt)}</div>
+          <div class="temp"> ${filterTable[3].main.temp.toFixed(0)}°</div>
+      </div>
+               
+      <div class="dayAndTemp">
+          <div class="day">${weekDay(filterTable[4].dt)}</div>
+          <div class="temp"> ${filterTable[4].main.temp.toFixed(0)}°</div>
+      </div>  `
+          })
```
Comment (jessnmo) [inline script.js:207]: Here we could use a forEach for the filtered table to loop through the week day by getting the dates (dt), and get the name of that date (i.e. Mon - Sun). 

Once you have the days, the correct container div .innerHTML += a para for day, a para for temperature. JS will generate div itself and give a class name

File: style.css [hunk 5]
```diff
@@ -0,0 +1,89 @@
+.header{
+    font-family: 'Montserrat', sans-serif;
```
Comment (jessnmo) [inline style.css:2]: you could create a 
*{
font-family: ... 
}

* means all, so you don't have to add font-family to everything

**Pull request closed without merging**